### PR TITLE
Fixes Salad Logo Avatar Selection Bug

### PR DESCRIPTION
**Pull request by jpark799**
need to wait on backend changes to handle `00000000-0000-0000-0000-000000000000` id to be merged in first

### CODE REVIEW

File: packages/web-app/src/modules/profile/ProfileStore.ts [hunk 1]
```diff
@@ -109,14 +111,17 @@ export class ProfileStore {
...
     }
-    if (id === '0') {
-      this.currentSelectedAvatar = this.selectedAvatar = SaladDefaultAvatar.id
-      return
-    }
     this.avatarError = undefined
-    this.selectedAvatar = this.submittingAvatar = this.avatars.find((avatar) => avatar.id === id)?.id
+    let avatarId: string | undefined
+    if (id === '00000000-0000-0000-0000-000000000000') {
+      this.selectedAvatar = SaladDefaultAvatar.id
```
Comment (seniorquico) [inline packages/web-app/src/modules/profile/ProfileStore.ts:117]: Just comparing this line to the one in the `else` branch... should this be like the following?

```suggestion
      this.selectedAvatar = this.submittingAvatar = SaladDefaultAvatar.id
```

**Comment by seniorquico** [comment]
@jpark799 The API changes have been pushed live.

File: packages/web-app/src/modules/profile/ProfileStore.ts (hunk 1 above)
Comment (jpark799) [reply packages/web-app/src/modules/profile/ProfileStore.ts:117]: ah you're exactly right. thank you for catching this.

**Comment by jpark799** [comment]
> @jpark799 The API changes have been pushed live.

yeah, i was working on this earlier today but it was erroring out and i told myself i'd come back to this after finishing my other PR to hook things up in the onboarding pages. hooking things up and addressing scott's comments unfortunately took longer than i had hoped. the error i was getting was because of the line you pointed out above and it's been fixed now. thank you

**Review by seniorquico — APPROVED** [review]

**Pull request merged**